docs/How-To-Guides/how-to-become-a-maintainer.md

```diff
@@ -0,0 +1,36 @@
+---
+sidebar_position: 3
+---
+
+# How to Become a ReactPlay Open Source Project Maintainer
+
+[ReactPlay](https://reactplay.io/) is an open-source community that welcomes volunteers to become maintainers. If you are interested in becoming a maintainer, here are the criteria and requirements you need to meet.
+
+## About the Role
+
+- It is a volunteer effort. 🙋‍♂️
+- You must have a love for open source and people. 💛
+- You may have to give 4-5 hours a week. ⏳
+
+## Criteria
+
+- Be a Contributor first. Make sure you have contributed to at least 2 issues to understand the ReactPlay ecosystem, processes, and accepting criteria for any contributions.
+- Make a short write-up explaining why do you want to take up this role? This write-up should be no more than 150 words.
+- Schedule a meeting with [https://zcal.co/tapasadhikary/15min](https://zcal.co/tapasadhikary/15min) for 15 mins to discuss your points.
```

Comment:
Can we just link the `Schedule a meeting with` instead of making the link as text?

Reply:
The `with` is redundant.